src/rustllvm/PassWrapper.cpp

```diff
@@ -188,7 +188,7 @@ LLVMRustCreateTargetMachine(const char *triple,
     }
 
     TargetOptions Options;
-    Options.PositionIndependentExecutable = PositionIndependentExecutable;
+    //Options.PositionIndependentExecutable = PositionIndependentExecutable;
```

Comment:
Right, this is one of the things mentioned above. I left it in, so I remember to fix that.